src/types/timestamp.rs

```diff
@@ -1296,6 +1390,40 @@ mod timestamp_tests {
         assert_eq!(12345678, super::first_n_digits_of(8, 123456789));
         assert_eq!(123456789, super::first_n_digits_of(9, 123456789));
     }
+
```

Comment:
🗺️ These unit tests had accidentally been placed in the testing module for `ion-c`'s timestamp code. They've been moved to the native impl testing module. You'll see the corresponding deletion immediately following.